devices/stm32/stm32f3-03-6_8.xml

```diff
@@ -133,46 +133,86 @@
       <channels instance="1">
         <channel position="1">
           <signal driver="adc" instance="1" name="dma"/>
-          <signal driver="tim" instance="17" name="ch1"/>
-          <signal driver="tim" instance="17" name="up"/>
+          <signal driver="tim" instance="17" name="ch1">
+            <remap position="12" mask="1" id="0"/>
+          </signal>
+          <signal driver="tim" instance="17" name="up">
+            <remap position="12" mask="1" id="0"/>
+          </signal>
           <signal driver="tim" instance="2" name="ch3"/>
         </channel>
         <channel position="2">
-          <signal driver="adc" instance="2" name="dma"/>
-          <signal driver="i2c" instance="1" name="tx"/>
-          <signal driver="spi" instance="1" name="rx"/>
+          <signal driver="adc" instance="2" name="dma">
+            <remap position="72" mask="3" id="2"/>
+          </signal>
+          <signal driver="i2c" instance="1" name="tx">
+            <remap position="70" mask="3" id="1"/>
+          </signal>
+          <signal driver="spi" instance="1" name="rx">
+            <remap position="64" mask="3" id="0"/>
+            <remap position="64" mask="3" id="3"/>
```

Comment:
`(position % 32) + 1` gives the registers number, here `CFGR3`. Same idea used for AFIOx in STM32F1 GPIO remappings.

Reply:
No clue why ST thinks it needs two mappings for this value. I'm not sure if this isn't a copy-paste error in the PDF.